wolfssl/wolfio.h

```diff
@@ -50,6 +50,12 @@
     #endif
 #endif
 
+/* CE Not always reliably detected. Define our own _WIN32_WCE as needed. */
+#if defined(_WIN32_WCE) || defined(WINCE) || defined(PocketPC)
+    #if !_WIN32_WCE
```

Comment:
Please change this to `#if !defined(_WIN32_WCE)`. Why add another macro `WINCE` or `PocketPC` here? Are those supposed to supplied by the compiler or optionally by the user?

Reply:
I will change this to only `_WIN32_WCE`. Neither that, nor `WINCE` have ever been auto-defined for me. I added them for likely completeness, although never actually tested.

The `PocketPC` was auto-added at new project time here. I manually added the WindowsCE.

![image](https://github.com/user-attachments/assets/eb4e6c24-84fa-4da4-a23b-d080b21fc038)